### ISSUE TITLE
Settings UI: `code` text not legible in high contrast themes

### ISSUE THREAD

**Issue by sharon-wang** [posit-dev/positron#8434]
#### Positron and OS details:

Positron Version: 2025.08.0 build 23
Code - OSS Version: 1.100.0
Commit: 90b4e5ca3b459c65f692c8573424467a1e9ad220
Date: 2025-07-04T03:34:51.515Z
Electron: 34.5.1
Chromium: 132.0.6834.210
Node.js: 20.19.0
V8: 13.2.152.41-electron.0
OS: Darwin arm64 24.5.0

#### Interpreter details:

N/A

## Describe the issue:

In the Settings UI, `code` text in setting descriptions are not legible when using either of the default high contrast themes.

https://github.com/user-attachments/assets/cd5c565b-96d1-4795-895d-455e99589ecc

## Steps to reproduce the issue:

1. Use either of the built-in Light or Dark High Contrast themes
2. Open the Settings UI and search for a setting that includes `code` elements in their description, e.g. `quarto.render.renderOnSaveShiny`
3. Notice that some of the text (`code` elements) appears to be missing

## Expected or desired behavior:

The full setting description should be legible when using either of the default high contrast themes.

## Were there any error messages in the UI, Output panel, or Developer Tools console?

N/A

**Comment by testlabauto** [comment]
### Verified Fixed
    Positron Version(s) : 2026.01.0-125
    OS Version          : OSX

### Test scenario(s) 

Code text is very visible now in high contrast themes.

### Link(s) to test cases run or created:
